packages/core/core/src/BundleGraph.js

```diff
@@ -518,7 +518,7 @@ export default class BundleGraph {
     let hash = crypto.createHash('md5');
     // TODO: sort??
     this.traverseAssets(bundle, asset => {
-      hash.update(asset.outputHash);
+      hash.update([asset.outputHash, asset.filePath].join(':'));
```

Comment:
Won't this break sharing cache between machines as this is an absolute path?

Not sure if this is something we wanna support.

Reply:
> Not sure if this is something we wanna support.

Will was working on a HTTP remote cache backend, so I would say yes.

Reply:
Yeah, we'll definitely need filepaths on the assets themselves to be relative to something like the project root.